docs/client-context-aware-configuration.md

```diff
@@ -295,7 +295,7 @@ Get the full config definition of your tenants configuration from superposition.
 ##### Funtion Definition
 
 ```
-getFullConfigStateWithFilter :: ForeignPtr CacClient -> Maybe String -> IO (Either Error Value)
+getFullConfigStateWithFilter :: ForeignPtr CacClient -> Maybe String -> Maybe String -> IO (Either Error Value)
```

Comment:
@mahatoankitkumar can we add the table explaining what each parameter is? I missed this the last time